coverband.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake'
   spec.add_development_dependency 'redis'
   spec.add_development_dependency 'test-unit'
-  spec.add_development_dependency 'rails', '~>5'
```

Comment:
with two Gemfiles which explicitly call out Rails, do we still need or want this? @kbaum

Reply:
Oh.. good catch! We don't need this in the gemspec anymore.